src/main/java/uk/ac/ebi/eva/pipeline/parameters/validation/ParametersValidatorUtil.java

```diff
@@ -28,6 +28,30 @@
  */
 public class ParametersValidatorUtil {
 
+    static void checkIsValidString(String stringToValidate,
+                                   String jobParametersName) throws JobParametersInvalidException {
+        checkIsNotNullOrEmptyString(stringToValidate, jobParametersName);
+        checkAsciiString(stringToValidate, jobParametersName);
+        checkSingleLineString(stringToValidate, jobParametersName);
+    }
+
+    static void checkAsciiString(String stringToValidate,
+                                 String jobParametersName) throws JobParametersInvalidException {
+        if (!stringToValidate.matches("\\A\\p{ASCII}*\\z")) {
+            throw new JobParametersInvalidException(
+                    String.format("%s in %s contains non ascii characters", stringToValidate, jobParametersName));
+        }
+    }
+
+    static void checkSingleLineString(String stringToValidate,
+                                      String jobParametersName) throws JobParametersInvalidException {
+        final String NEW_LINE = System.getProperty("line.separator");
```

Comment:
This should be valid:

```
input.study.title="1000 Genomes Phase 3\nVersion 5"
```

But the one below should fail. Maybe Spring will raise an error automatically because one of the parameters is not a key-value pair.

```
input.study.title=1000 Genomes Phase 3
Version 5
```

But if the same multi-line string is surrounded by quotes it should be fine. Does the pipeline work in a case like this?

```
input.study.title="1000 Genomes Phase 3
Version 5b"
```